cidc_api/models/models.py

```diff
@@ -1265,6 +1265,136 @@ def get_summaries(session: Session) -> List[dict]:
                 trial_metadata,
                 jsonb_array_elements(metadata_json#>'{assays,cytof_e4412}') batches,
                 jsonb_array_elements(batches->'participants') participant
+            union all
+            select
+                trial_id,
+                'cytof_analysis' as key,
+                case
+                    when sample->'output_files' is not null then 1 else 0
+                end as value
+            from
+                trial_metadata,
+                jsonb_array_elements(metadata_json#>'{assays,cytof_e4412}') batches,
+                jsonb_array_elements(batches->'participants') participant,
+                jsonb_array_elements(participant->'samples') sample               
+        """
+
+        cytof_10021_analysis_subquery = """
+            select
+                trial_id,
+                'cytof_analysis' as key,
+                case
+                    when record->'output_files' is not null then 1 else 0
+                end as value
+            from
+                trial_metadata,
+                jsonb_array_elements(metadata_json#>'{assays,cytof_10021}') batch,
+                jsonb_array_elements(batch->'records') record
+        """
+
+        wes_analysis_subquery = """
+            select
+                trial_id,
+                'wes_analysis' as key,
+                2 * jsonb_array_length(metadata_json#>'{analysis,wes_analysis,pair_runs}') as value
```

Comment:
this will also count anything with only a tumor/normal pairing, as that manifest generates minimal wes_analysis pair_runs
seems like we would want to check if the other subfields are not null